.travis/scripts.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
```

Comment:
Let's use a proper bash shebang here for portability: `#!/usr/bin/env bash`. Then on the next line `set -e`.